### ISSUE TITLE
No module named 'torch._dynamo'

### ISSUE THREAD

**Issue by iiijasdf** [Vanessik/HAAR#4]
Hi, first thanks for such an amazing work, but I may need some help to get my inference available to run......

I followed the instruction to configurate my conda environment and tried the example code showed in README, but I got the following error: 

No module named 'torch._dynamo'

I checked some blogs and they suggested me upgrade my CUDA to >= 11.8 to install pytorch 2.0.*, then I can import torch._dynamo normally, but when I tried to upgrade pytorch, so many incompatible errors came out. 

I wonder that if there is any other faster solution to get the inference code running? Thanks!

**Comment by iiijasdf** [comment]
Finally I made it work on my environment. For somebody who is still stuck in the configuration of the env, I post my process here nad hope it help. 

1. I created a blank environment with python 3.9, then installed pytorch 2.0.1 and CUDA. 
2. Then I ran this code for a few times:
    `python infer.py --exp_name infer_haar --conf ./configs/infer.json --ckpt_path ./pretrained_models/haar_prior/haar_diffusion.pth --cfg_scale 1.5 --save_latent_textures --save_guiding_strands --save_upsampled_hairstyle --upsample_resolution 64 --n_samples 10 --hairstyle_description "long ponytail hairstyle"`
   and installed every module it hinted. I also exported my conda environment and pip freeze files FYI. 
   [HAARTest.zip](https://github.com/user-attachments/files/16230606/HAARTest.zip)
3. I also downloaded the blip_base_uncased model manually because automatic cloning seems failed to download this model. Then I changed some codes in the submodule blip2 (in submodules/LAVIS/lavis/models/blip2_models/blip2.py), searched every 
'"bert-base-uncased"' and replace with 'BERT_LOCAL_PATH, local_files_only=True'. 
4. Finally I got the example code running (although with a poor result due to low amount of dataset maybe). 
![image](https://github.com/user-attachments/assets/e0981c51-fd2d-4a4c-b5a3-b5eb3d1bc6ea)

The main reason of this failure is that dynamo starts its support since pytorch 2.0.*, but the environment suggests 1.13.0. Since reinstall the right version of pytorch caused too many conflicts of package version, so I create a new env from the beginning. 

Hope this helps.
